src/components/TextBox/__stories__/Template.stories.tsx

```diff
@@ -0,0 +1,6 @@
+import { ComponentStory } from '@storybook/react'
+import UncontrolledTextBox from './UncontrolledTextBox'
+
+export const Template: ComponentStory<typeof UncontrolledTextBox> = args => (
+  <UncontrolledTextBox {...args} />
+)
```

Comment:
Can you import `UncontrolledTextBox` content in here instead and remove the file? We don't want to have such snippet of code as it doesn't represent how to use the component:

<img width="319" alt="Screenshot 2022-12-01 at 15 39 43" src="https://user-images.githubusercontent.com/15812968/205081015-aa602a44-1e53-4fb8-9b05-9e2ae73a311a.png">